<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelPicturePreview</class>
 <widget class="QWidget" name="PanelPicturePreview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,1">
     <item>
      <widget class="QCheckBox" name="checkBoxContent">
       <property name="text">
        <string>Show available content</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,1">
       <item>
        <widget class="PanelSuperList" name="widgetPanelIDs" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
         <item>
          <widget class="QTreeView" name="treeViewAvailableContent">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="pushButtonMove">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../ressources.qrc">
               <normaloff>:/icons/Ressources/big_arrow_left.png</normaloff>:/icons/Ressources/big_arrow_left.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonAdd">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
         <item>
          <widget class="QWidget" name="widgetPreview" native="true"/>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Options</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PanelSuperList</class>
   <extends>QWidget</extends>
   <header>panelsuperlist.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
